printer_mods/chrisrgonzales/270_degree_hinge/README.md

```diff
@@ -0,0 +1,121 @@
+is shallow as you may risk of pushing through. The backplates have a chamber for more recessed look, but can be flipped 
+if you have longer hardware.
+
+Take it slow when installing heat inserts as its very easy to push them through and cause deforming the face of the hinge.
+Set temp on iron lower than used on most other heat inserted parts as this gives you more time not to press through. Also 
+you may have to use the side of your iron's tip as not to puncture all the way.
+
+For placement of drilling, you can install the hinges in place, and tape or use panel clips from sides temporarily to hold
+front doors and tape outline of hinge. Then remove panel and hinge place hinge where tape is outline then mark holes and drill.
+
+TIP: When installing with tape, is to mount the panels with the same corner clips used on the sides and top, you can steal 
+a couple from each panel if you already have your side setup. Anyways use the corner clips to position how the doors 
+should be position when closed. Install the hinges on the side and at the desired positions. Now apply double-side adhesive 
+to the hinges, then swing the hinge inwards touching the panel and secure them. Once you feel the adhesive is secure, remove 
+the corner clips
+
+## CAD Files
+
+The CAD files are parametric! When opened in Fusion 360, editing the `thickness` parameter will change the spacing and 
+geometry for to allow the hinge full 270 articulation. Enter in thickness of your panel and foam, I would recommend 
+accounting for the compression in your foam. Example 3 mm panel with 3 mm foam, typical 6 mm would be entered, but possible
+that 5.8 mm might be a better choice to give some compression for a seal. If you are unsure, printing a single bracket and
+testing fitment and offset would be ideal.
+
+If you would like to use this with tape only you may adjust the `faceThickness` parameter to be thinner but wouldn't 
+suggest going smaller than 3 mm. You will also need to remove or extrude flush the holes for the brass inserts. Also the
+`faceWidth` parameter may adjusted to be smaller width. Would not go lower than 27 mm for this parameter.
+
+The logo can be removed if needed. If using Fusion 360 with history, step back two actions and logo will be removed. 
+Otherwise removal of chamfer and extrude flush will be needed.
+
+```
+thickness      : panel + foam depth in mm (default 4 mm)
+faceThickness  : depth of the face portion of hinge no including panel + foam (default 5 mm)
+faceWidth      : width of facing hinge from edge of extrusion to edge of center of printer. Not actual width of hinge default (34.9 mm)
+
+```
+
+### Questions / Suggestions
+If you have any questions or suggestions feel free to contact me on Discord [chrisrgonzales#0731](https://discord.com/users/chrisrgonzales#0731)
+
+![](Images/installed-close.jpg)  |  ![](Images/installed-180.jpg) |  ![](Images/installed-270.jpg)
+ --- | --- | ---
+ 0° |  180° |  270°
```

Comment:
For 4 mm and 6 mm you include both drilled and tape STLs, but for 3 mm and 5 mm you only include drilled - should 3 and 5 mm tape versions also be included?